### PR TITLE
{phys}[gompi/2022a] XBeach v20230831

### PR DESCRIPTION
**Pull request by WilleBell**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/x/XBeach/XBeach_fix-gfortran-compilation.patch
Comment (WilleBell) [inline easybuild/easyconfigs/x/XBeach/XBeach_fix-gfortran-compilation.patch:1]: patch from [https://github.com/openearth/xbeach/issues/2](https://github.com/openearth/xbeach/issues/2)

**Comment by WilleBell** [comment]
Patch from [https://github.com/openearth/xbeach/issues/2](https://github.com/openearth/xbeach/issues/2)

**Comment by WilleBell** [comment]
Test report by @WilleBell
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in total)
node311.hydra.os - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 3.6.8
See https://gist.github.com/WilleBell/5392d96aa7224a92477e1b4557aac00c for a full test report.

**Comment by WilleBell** [comment]
Test report by @WilleBell
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in total)
node308.hydra.os - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 3.6.8
See https://gist.github.com/WilleBell/f041b919ba3f28955df58d0f5f62ccfd for a full test report.

**Comment by WilleBell** [comment]
Test report by @WilleBell
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in total)
node308.hydra.os - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 3.6.8
See https://gist.github.com/WilleBell/f8855ff4c73abbf042a8b8c3210c63ad for a full test report.

**Comment by WilleBell** [comment]
Test report by @WilleBell
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node308.hydra.os - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 3.6.8
See https://gist.github.com/WilleBell/810e7e108eea2495b78390e526993f19 for a full test report.

**Review by smoors — APPROVED** [review]
lgtm

**Comment by smoors** [comment]
Going in, thanks @WilleBell!

**Pull request merged**